src/components/vuestic-components/vuestic-modal/VuesticModal.vue

```diff
@@ -108,7 +112,7 @@
       },
       cancel () {
         this.$emit('cancel')
-        this.show = false
+        void(this.okToClose ? this.show = false : null)
```

Comment:
Isn't it suppose to go to `close` method?

```
      close () {
        if (this.okToClose) {
          this.show = false
        }
      }
```

P.S. Run `npm run lint` to lint your changes locally